lib/season_stat.rb

```diff
@@ -92,9 +91,10 @@ def team_win_percentage(team_id, game_type, season)
     ((total_wins / total_games) * 100).round(2)
   end
 
-  def get_regular_percents(game_type)
```

Comment:
refactored this helper method out of season too, it was redundant and I was accomplishing this in the get_team_info method